python_modules/dagster/dagster_tests/core_tests/storage_tests/test_assets.py

```diff
@@ -406,6 +408,33 @@ def gen_everything():
         ]
 
 
+@asset_test
+def test_get_observation(asset_aware_context):
```

Comment:
I wonder if we should just move these asset tests into the event log storage test suite.

This was broken out back when only some storages had asset support, but we've had coverage for a few major releases now.

Reply:
Yes, it makes sense to move these tests to the test suite. I will break this out in a separate PR.